Sources/StrongContractResponse/StrongContractResponse.swift

```diff
@@ -66,23 +67,22 @@ public struct Request<Payload: Codable, Response: Codable> {
     public func task(
         autoResume: Bool = true,
         expressive: Bool = false,
-        assertHasAccessToken: Bool = true,
         payload: Payload,
         passResponse: @escaping PassResponse,
         errorHandler: ErrorHandler? = nil
     ) -> URLSessionDataTask? {
-        if assertHasAccessToken {
-            URLQueryItem.assertHasToken()
-        }
```

Comment:
Please add back the asserts but for the header

Reply:
Done